### PR TITLE
[MM-13280] fix uncaught js error when issuing a slash command.

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
is also happens in the search box and other components that use the suggestion box 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-13280

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

File: components/suggestion/suggestion_box.jsx [hunk 1]
```diff
@@ -498,8 +498,8 @@ export default class SuggestionBox extends React.Component {
 
             if (handled) {
                 this.setState({
-                    presentationType: provider.presentationType(),
-                    allowDividers: provider.allowDividers(),
+                    presentationType: provider.presentationType,
+                    allowDividers: provider.allowDividers,
```
Comment (saturninoabril) [inline components/suggestion/suggestion_box.jsx]: Not sure about this fix.  `EmoticonProvider` don't have implementation of `presentationType` and `allowDividers`, so I guess we'll have to add those there.

Comment (cpanato) [reply components/suggestion/suggestion_box.jsx]: will be `undefined`, I'm not sure as well. I can add the implementation in the missing ones like the commandprovider and emoiticonprovider, but i will need to know which values to add.

Comment (saturninoabril) [reply components/suggestion/suggestion_box.jsx]: With the way we're currently showing it, I'm guessing it's like:
- EmoticonProvider - "allowDividers: false" and "presentationType: text"
- CommandProvider - "allowDividers: true" and "presentationType: text"

Comment (cpanato) [reply components/suggestion/suggestion_box.jsx]: will update, thanks so much for the review @saturninoabril

**Comment by cpanato** [comment]
closing in favor of https://github.com/mattermost/mattermost-webapp/pull/2139

**Pull request closed without merging**